src/clj_irods/jargon.clj

```diff
@@ -59,7 +59,7 @@
 (defn- get-metadata*
   [irods path & {:keys [known-type]}]
   (->> [path ::get-metadata]
-       (cache/cached-or-do (:cache irods) #(metadata/get-metadata @(:jargon irods) path))))
+       (cache/cached-or-do (:cache irods) #(metadata/get-metadata @(:jargon irods) path :known-type known-type))))
```

Comment:
Actually passing the known type down to clj-jargon like I should have originally...